corehq/apps/es/migration_operations.py

```diff
@@ -45,7 +48,10 @@ class CreateIndex(BaseElasticOperation):
 
     serialization_expand_args = ["mapping", "analysis"]
 
-    def __init__(self, name, type_, mapping, analysis, settings_key, comment=None, es_versions=[]):
+    def __init__(
+            self, name, type_, mapping, analysis, settings_key,
+            comment=None, es_versions=[], creation_checks=False
+    ):
```

Comment:
Looks like all the other parameters for this are documented. Could we add a documentation line for `creation_checks`?